### PR TITLE
Feat/noughts and crosses

### PR DESCRIPTION
**Pull request by ghost**
Added example noughts and crosses game that uses useEffect and useState

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -23,12 +24,18 @@ function App() {
...
-          <Rotate>
-            {" "}
-            <BoringPie />
-          </Rotate>
+          <div>
+            <Rotate inline>
+              {" "}
+              <BoringPie />
+            </Rotate>
+          </div>
```
Comment (FredsW47) [inline src/App.js:32]: Ahaha does my pie look great now?

File: src/components/molecules/NoughtsAndCrosses/NoughtsAndCrosses.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    null,
+    null,
+  ]);
+  const [currentPlayer, setCurrentPlayer] = useState("X");
+  const [winner, setWinner] = useState(null);
+
+  function playTurn(squareIndex) {
+    if (winner === null) {
+      if (gameState[squareIndex] === null) {
+        var newGameState = [...gameState];
```
Comment (FredsW47) [inline src/components/molecules/NoughtsAndCrosses/NoughtsAndCrosses.js]: Do you think we should add a comment saying what the triple ...  does?

File: src/components/molecules/NoughtsAndCrosses/NoughtsAndCrosses.js [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+      if (possibleDraw === true) {
+        setWinner("DRAW");
+      }
+    }
+  }, [gameState]);
+
+  return (
+    <div className="boardContainer">
+      <div className="gameBoard">
+        {gameState.map((squareValue, squareIndex) => {
```
Comment (FredsW47) [inline src/components/molecules/NoughtsAndCrosses/NoughtsAndCrosses.js]: This is cool returning a div like this!

File: src/components/molecules/NoughtsAndCrosses/NoughtsAndCrosses.js (hunk 2 above)
Comment (ghost) [reply src/components/molecules/NoughtsAndCrosses/NoughtsAndCrosses.js]: Good point, should add comments to explain different sections

File: src/components/molecules/NoughtsAndCrosses/NoughtsAndCrosses.js (hunk 3 above)
Comment (ghost) [reply src/components/molecules/NoughtsAndCrosses/NoughtsAndCrosses.js]: 😎

File: src/App.js (hunk 1 above)
Comment (ghost) [reply src/App.js:32]: 100% pie improvement

File: src/components/molecules/NoughtsAndCrosses/NoughtsAndCrosses.js (hunk 2 above)
Comment (ghost) [reply src/components/molecules/NoughtsAndCrosses/NoughtsAndCrosses.js]: Added comments

**Pull request merged**